en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/migration-guide-from-api-microgateway.md

```diff
@@ -0,0 +1,130 @@
+
+### Choreo Connect v1.2.0
+
+Choreo Connect accepts the following configurations for retry policy.
+
+``` yaml
+x-wso2-production-endpoints:
+    urls:
+      - https://localhost:2380/v3
+    advanceEndpointConfig:
+      retryConfig:
+        count: 3
+        statusCodes:
+          - 504
+
+```
+
+- **`retryConfig`** given under prod or sandbox endpoint only has `count` and `statusCode`. 
+- Envoy uses a fully jittered exponential back-off algorithm for retries. Therefore the interval between the retries is always a random number. More information available in [here](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/router_filter#x-envoy-max-retries).
+- A param called **`backOffFactor`** does not exist in the envoy’s retry interval equation, therefore is not expected in the config. 
+- **`intervalInMillis`** can only be configured via envoy route configs (not configurable via header) which is common to both prod and sandbox. Thus, we have made **`intervalInMillis`** configurable via a parameter **`BaseIntervalInMillis`** in the `config.toml` which is the base interval for the Envoy's exponential retry back off algorithm.
+- An upper bound can be set for the retry count taken from all API definitions. It can be set via **`MaxRetryCount`** set in `config.toml`. When the configured retry count in the API definition is larger than the `MaxRetryCount`, `MaxRetryCount` will be assigned as the count.
+- Only a range of (401 - 598) status codes can be used to enable retry. If a status code out of this range is given in an API definition, that status code will get dropped when deploying the API. If this leads to an empty array of status codes, the default status codes configured in `config.toml` will be used. 
+
+## Moving APIs from Ballerina MGW to Choreo Connect
+
+### Standalone Gateway
+
+There is no straightforward way to migrate APIs from Ballerina MGW to Choreo Connect. Users need to create the APIs from scratch using the OpenAPI/Swagger definitions.
+
+Using the apictl command line tool, users can generate apictl projects as follows.
+
+``` bash
+apictl init project-1 --oas <Path_to_OpenAPI_File>
+```
+
+Step by step guide includes in [here]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-rest-api-in-choreo-connect/#choreo-connect-as-a-standalone-gateway).
+
+### Choreo Connect with WSO2 API Manager 4.2.0
+
+Choreo Connect v4.2.0 is only compatible with WSO2 API Manager 4.2.0. Hence you need to upgrade to 4.2.0.
+
+1. Upgrade WSO2 API Manager version to 4.2.0. 
+
+    A detailed explanation regarding the above includes in  [here]({{base_path}}/install-and-setup/upgrading-wso2-api-manager/320-to-410/upgrading-from-320-to-410/).
+
+2. Connect Choreo Connect microgateway to API Manager 4.2.0 as a gateway.
```

Comment:
@samgnaniah @pubudu538 
Please check the version and the URL of step one in here.